src/components/SearchForm.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import "../blocks/SearchForm.css";
+
+export default function SearchForm({ handleSearchSubmit }) {
+  const [value, setValue] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    handleSearchSubmit(value);
+  };
+
+  return (
+    <form className="search" onSubmit={handleSubmit}>
+      <label htmlFor="search" className="search__label">
+        <input
+          id="search"
+          type="text"
+          className="search__input"
+          placeholder="Enter topic"
+          maxLength="20"
+          onChange={(e) => setValue(e.target.value)}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add the `value` attribute to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components